### DISCUSSION TITLE
integration with the serverless framework

### DISCUSSION THREAD

**Discussion by ianzone** [JakePartusch/serverlessui#44] [Ideas]
Hi there, thanks for creating this interesting tool, just wondering is there any plan to wrap this project as a serverless component? Since there are a lot of people using the serverless framework.

**Comment by JakePartusch** [comment]
Hi @ianzone 👋 , no plans at the moment — this tool relies pretty heavily on the aws-cdk. So, it would be a relatively large lift to make it compatible with the serverless framework.
